framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -1199,7 +1201,7 @@ describe('BaseCrossChainUpdateCommand', () => {
 					status: ccmStatus,
 					sendingChainID: defaultCCM.receivingChainID,
 					receivingChainID: defaultCCM.sendingChainID,
-					fee: context.ccm.fee - BigInt(100) * MIN_RETURN_FEE,
+					fee: BigInt(99000000000),
```

Comment:
maybe we can break down the calculation instead of a big number without context, or add a comment related to it.
Also below in other places

Reply:
I think this is the logic of bounce itself, you have `ccm.fee` and then you deduct minimum fee `getMinReturnFeePerByte(ccm.sendingChainID) * len(encode(crossChainMessageSchema, ccm))` ,i think this is one of the logics we should check properly within bounce, the problem with arbitrary number is it can pass test for invalid cases as well.
and this comment will not help much as it still doesn't explain how we came up with that number.
`// See LIP 45 and getMinReturnFeePerByte implementation for fee calculation breakdown`
you can simply use below I think, very simple change but it could help to maintain tests,
`(context.ccm.fee - (MIN_FEE_PER_BYTE * codec.encode(ccmSchema, bouncedCCM)))`

Reply:
Comment says see LIP 45 which explains this calculation in detail

Reply:
created issue related to it https://github.com/LiskHQ/lisk-sdk/issues/8195

Reply:
As we already discussed in great length in the retro, I mentioned I would fix it in a new issue.